examples/run_nutation.py

```diff
@@ -61,84 +61,65 @@ def API_sender(value):
     API_sender(B0)
 #}}}
 date = datetime.now().strftime('%y%m%d')
-output_name = 'TEMPOL_capProbe'
-adcOffset = 25
-carrierFreq_MHz = 14.896101
+output_name = 'TEMPOL_capillary_probe_nutation_1'
+ph1_cyc = r_[0,2]
+ph2_cyc = r_[0,2]
+nPhaseSteps = len(ph1_cyc)*len(ph2_cyc)
+adcOffset = 39
+carrierFreq_MHz = 14.896010
 tx_phases = r_[0.0,90.0,180.0,270.0]
 amplitude = 1.0
 nScans = 1
 nEchoes = 1
 phase_cycling = True
-if phase_cycling:
-    nPhaseSteps = 8
-if not phase_cycling:
-    nPhaseSteps = 1
 # NOTE: Number of segments is nEchoes * nPhaseSteps
-deadtime = 10.0
-repetition = 12e6
+deadtime_us = 10.0
+repetition_us = 10e6
 SW_kHz = 24.0
 nPoints = 1024
-acq_time = nPoints/SW_kHz # ms
-tau_adjust = 0.0
-tau = 1000
-print("ACQUISITION TIME:",acq_time,"ms")
-print("TAU DELAY:",tau,"us")
+acq_time_ms = nPoints/SW_kHz # ms
+tau_adjust_us = 0.0
+tau_us = deadtime_us + acq_time_ms*1e3*0.5 + tau_adjust_us
+print("ACQUISITION TIME:",acq_time_ms,"ms")
+print("TAU DELAY:",tau_us,"us")
```

Comment:
these seem to be parameter changes -- use the same parameters (i.e. hard-coded settings) as master.

Reply:
(we don't want irrelevant changes in git)

Reply:
line 82 seems to be relevant